iroh-p2p/Cargo.toml

```diff
@@ -53,6 +53,7 @@ features = [
   "noise",
   "yamux",
   "tcp",
+  "quic",
```

Comment:
should we alpha sort these at some point?

Reply:
PRs welcome :P